taccsite_cms/templates/guides/portal_technology.html

```diff
@@ -5,36 +5,40 @@ <h1>Portal Technology Stack</h1>
 
   <h2>Portal Overview</h2>
 
-  <p>CEP, Core Experience Portal, is the core engine for all TACC portal projects. The primary goal of the Core Portal project is to establish a codebase that can be used as a springboard for all future portal projects undertaken by TACC. By establishing a common codebase for all portal efforts, we can better maintaining alignment between the core capabilities and technologies supported across all TACC portals. There will be unique requirements in some portal projects, but CEP should provide an "out-of-the-box" framework for rapidly deploying a new portal project with all the common capabilities already in place and compliant with current best practices and conventions at TACC.</p>
+  <p>The TACC CEP, short for Core Experience Portal, is the core engine for all TACC portal projects. The primary objective for the Core Experience Portal is to establish and grow a codebase that can be used as a common baseline for all future portal projects undertaken at TACC. By cultivating this common codebase for all of our portal projects, we maintain better alignment between the core capabilities and technologies supported across all of the TACC portals. While some projects do have unique requirements, CEP provides an "out-of-the-box" framework for rapidly deploying a ready-to-use project infrastructure that includes all the common TACC portal capabilities pre-configured and in compliance with TACC's current best practices.</p>
 
-  <h3>Common Portal Capabilities:</h3>
+  <h3>Common Portal Capabilities</h3>
 
```

Comment:
The whole Common Portal Capabilities list feels a little off. Later I was more diligent about distinguishing between types of systems, etc. I am not sure this section is consistent with the rest of my edits. Thoughts?

Reply:
Well… mentioning Elastic Search is unique. No other bullet in this list mentions a software. And all other proper nouns are suggestive of a component using non-software terms.

Reply:
I am a little confused on the state of our portal search engine - are we still using ES or have we transitioned to Google for all things search now?

Reply:
Suggestions, drop Elastic Search here, or move it to "Environment" and reword this to just be—

> **Search**

—or—

> An **Integrated Search**

Reply:
We have not transitioned to Google for CMS search. That transition comes piecemeal. Elastic Search is still used for "behind-the-login" search.

Reply:
It's fine as is. Sorry for any brain belaboring I caused. I haven't heard @jarosenb mention abandoning Elastic Search for anything other than CMS.